packages/cubejs-server-core/src/core/server.ts

```diff
@@ -115,6 +115,8 @@ export class CubejsServerCore {
   public constructor(opts: CreateOptions = {}, protected readonly systemOptions?: SystemOptions) {
     optionsValidate(opts);
 
+    this.coreServerVersion = version;
```

Comment:
I think we can drop this version from the property because we get this version from package.json. m?